### PR TITLE
Added Optimism Hashflow Dex

### PR DESCRIPTION
**Pull request by ARDev097**
Added trades data for optimism hashflow dex using macro approach.

### CODE REVIEW

File: models/_sector/dex/trades/optimism/platforms/mummy_finance_optimism_base_trades.sql
Comment (Hosuke) [inline models/_sector/dex/trades/optimism/platforms/mummy_finance_optimism_base_trades.sql:1]: This PR contains `mummy_finance_optimism.base_trades` spell as well.

File: models/hashflow/optimism/hashflow_optimism_crosschain_trades.sql [hunk 1]
```diff
@@ -0,0 +1,104 @@
+{{
+    config(
+        alias = 'cross_chain_trades'
+        ,schema = 'hashflow_optimism'
+        ,partition_by = ['block_date']
+        ,materialized='incremental'
+        ,incremental_strategy = 'merge'
+        ,unique_key = ['block_date', 'source_chain', 'tx_hash']
+        ,post_hook='{{ expose_spells(\'["optimism"]\',
```
Comment (Hosuke) [inline models/hashflow/optimism/hashflow_optimism_crosschain_trades.sql:9]: It seems that `hashflow_optimism.cross_chain_trades` is relatively independent, as it is not included in `dex.trades` nor `dex_aggregator.trades`.

File: models/dex/dex_info.sql [hunk 2]
```diff
@@ -90,6 +90,7 @@ FROM (VALUES
     , ('mauve', 'Mauve', 'Direct', 'mauve_org')
     , ('xchange', 'X7 Finance', 'Direct', 'X7_Finance')
     , ('mento', 'Mento', 'Direct', 'mento')
+    , ('hashflow', 'Hashflow', 'Direct', 'hashflow')
```
Comment (Hosuke) [inline models/dex/dex_info.sql:93]: Defined in https://github.com/duneanalytics/spellbook/blob/7f03125cbb8a2100bb4c924ddd14dd580702909c/models/dex/dex_info.sql#L35

```suggestion
```

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @ARDev097 

[Sample hashflow_optimism.base_trades](https://dune.com/queries/3292138)
[Sample hashflow_optimism.crosschain_trades](https://dune.com/queries/3292152)

**Review by jeff-dude — APPROVED** [review]
works for me 👍 

tagging @BroderickBonelli as an fyi, due to name change on `crosschain` spells that will break queries on the app. i'm not sure if you two are working together, so playing it safe.

**Pull request merged**